src/sample/stuListController.java

```diff
@@ -313,6 +316,11 @@ public void updatebothlist() {
   public void courselist() {
     if (courseList.getSelectionModel().getSelectedItem() != null) {
 
+      //Clear the Course Details
+      courseTitle.setText("");
```

Comment:
Why need to clear the text🤔 it works fine even though you didn't clear the text🤔

Reply:
In case the Course Details file cannot display. So it will display empty instead of the previous selected course. IMO, display empty is better than  wrong info. Btw the file cannot display has a very small chance.